setup.cfg

```diff
@@ -0,0 +1,26 @@
+[bumpversion]
```

Comment:
We shouldn't use `bumpversion`, that project is no longer maintained. I suggest we use `setuptools_scm` instead, which works pretty well with Travis deployment (@williamjamir knows the details, feel free to ask him).